src/responder/kcm/kcm.c

```diff
@@ -48,6 +50,43 @@ static int kcm_responder_ctx_destructor(void *ptr)
     return 0;
 }
 
+static errno_t kcm_renewals_init(struct tevent_context *ev,
+                                 struct resp_ctx *rctx,
+                                 struct kcm_ctx *kctx,
+                                 struct krb5_ctx *krb5_ctx,
+                                 time_t renew_intv,
+                                 bool *_renewal_enabled)
+{
+#ifndef HAVE_KCM_RENEWAL
+    return EOK;
+#else
+    errno_t ret;
+
+    ret = kcm_get_renewal_config(kctx, &krb5_ctx, &renew_intv);
+    if (ret == ENOTSUP) {
+        DEBUG(SSSDBG_TRACE_FUNC, "TGT Renewals support disabled\n");
+        return EOK;
+    } else if (ret != EOK) {
+        DEBUG(SSSDBG_FATAL_FAILURE, "Unable to read TGT renewal configuration [%d]: %s\n",
+                                    ret, sss_strerror(ret));
+        return ret;
+    }
+
```

Comment:
It might be good to print the config here so we can see it in the logs.

Reply:
print the config? Sorry can you elaborate

Reply:
Print the option values that you read.

Reply:
I printed the values being set in `kcm_set_options`

Reply:
Fixed.